replica/database.hh

```diff
@@ -1129,10 +1127,15 @@ public:
 
     // Add sstable to main set
     void add_sstable(sstables::shared_sstable sstable);
+    // Add sstable to maintenance set
+    void add_maintenance_sstable(sstables::shared_sstable sst);
 
     const lw_shared_ptr<sstables::sstable_set>& main_sstables() const;
     void update_main_sstables(lw_shared_ptr<sstables::sstable_set> new_set);
 
+    const lw_shared_ptr<sstables::sstable_set>& maintenance_sstables() const;
+    void update_maintenance_sstables(lw_shared_ptr<sstables::sstable_set> new_set);
+
```

Comment:
ditto, rename to `set_maintenance_sstables`

Reply:
Will do.